static/app/views/replays/replayTable.tsx

```diff
@@ -0,0 +1,193 @@
+      projects: selection.projects,
+      query: `(title:sentry-replay-event OR event.type:error) AND (${query})`,
+    };
+
+    if (selection.datetime.period) {
+      eventQueryParams.range = selection.datetime.period;
+    }
+    return EventView.fromNewQueryWithLocation(eventQueryParams, location);
+  };
+
+  return (
+    <DiscoverQuery
+      eventView={getEventView()}
+      location={location}
+      orgSlug={organization.slug}
+    >
+      {data => {
+        const dataEntries = data.tableData
+          ? indexReplayEntriesById(
+              data.tableData?.data as ReplayDurationAndErrors[],
+              'replayId'
+            )
+          : null;
+        return replayList?.map(replay => {
+          return (
+            <Fragment key={replay.id}>
+              <ReplayUserBadge
+                avatarSize={32}
+                displayName={
+                  <Link
+                    to={`/organizations/${organization.slug}/replays/${generateEventSlug({
+                      project: replay.project,
+                      id: replay.id,
+                    })}/`}
+                  >
+                    {replay['user.display']}
+                  </Link>
+                }
+                user={{
+                  username: replay['user.display'],
+                  id: replay['user.display'],
+                  ip_address: replay['user.display'],
+                  name: replay['user.display'],
+                  email: replay['user.display'],
+                }}
+                // this is the subheading for the avatar, so displayEmail in this case is a misnomer
+                displayEmail={getUrlPathname(replay.url) ?? ''}
+              />
+              {isScreenLarge && (
+                <StyledPanelItem>
+                  <ProjectBadge
+                    project={
+                      projects.find(p => p.slug === replay.project) || {
+                        slug: replay.project,
+                      }
+                    }
+                    avatarSize={16}
+                  />
+                </StyledPanelItem>
+              )}
+              <StyledPanelItem>
+                <TimeSinceWrapper>
+                  {isScreenLarge && (
+                    <StyledIconCalendarWrapper color="gray500" size="sm" />
+                  )}
+                  <TimeSince date={replay.timestamp} />
+                </TimeSinceWrapper>
+              </StyledPanelItem>
+              {data.tableData && dataEntries ? (
+                <React.Fragment>
+                  <React.Fragment>
+                    <StyledPanelItem>
+                      <Duration
+                        seconds={
+                          dataEntries[replay.id]
+                            ? dataEntries[replay.id]['equation[0]']
+                            : 0
+                        }
+                        fixedDigits={2}
+                        abbreviation
+                      />
```

Comment:
Lets use `exact` to output `1min 58s 400ms` instead of something like `1.98min`
And then wrap with `Math.floor` to drop the `400ms` part
And then `|| 1` so that if the vlaue is between 0ms and 999ms, we show `1 second` instead.

```suggestion
                      <Duration
                        seconds={Math.floor(
                          dataEntries[replay.id]
                            ? dataEntries[replay.id]['equation[0]']
                            : 0
                        ) || 1}
                        abbreviation
                        exact
                      />
```